llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -1523,9 +1523,12 @@ unsigned RISCVInstrInfo::getInstSizeInBytes(const MachineInstr &MI) const {
   case TargetOpcode::PATCHPOINT:
```

Comment:
The title says `Stackmap/Statepoint/Patchpoint`, but we don't support them all here?

Reply:
This pull request only adds the feature where `Statepoint` and `Patchpoint` are treated as call instructions. Originally, this pull request was included in https://reviews.llvm.org/D123496, but to simplify the review, it was split in two parts `without target` and `with target`.

I will change the title to make this feature clearer and remove `Stackmap` as it is not modified by this pull request.